main/views.py

```diff
@@ -278,16 +280,40 @@ def add_searched_services():
 @app.route('/create_specialist', methods=['POST'])
 @login_required
 def create_specialist():
+    """
+    Func for Specialist creation
+    Calling by ajax
```

Comment:
`calling by` is not a proper form in English.
`Called by` is.
Also, usually you don't specify how the function is used, unless there is a specific reason.